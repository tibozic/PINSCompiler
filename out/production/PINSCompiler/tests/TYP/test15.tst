var sez: arr[10] integer;
fun get(i: integer): string = sez[i]
