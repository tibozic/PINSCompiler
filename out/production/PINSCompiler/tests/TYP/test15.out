[2:1-2:37]: ERROR: Function return type doesn't match function body return type: `str`!=`int`
